### ISSUE TITLE
pgadmin: 'psycopg2.extensions.Column' object has no attribute '_asdict'

### ISSUE THREAD

**Issue by ludydoo** [CrunchyData/postgres-operator#3409]
## Overview

Opening pgadmin and connecting to the database from the web ui throws an error `'psycopg2.extensions.Column' object has no attribute '_asdict'`. Network manager shows many failed request with this error as well

## Environment

Please provide the following details:

- Platform: `Openshift` 
- Platform Version: `4.11.3`
- PGO Image Tag: `5.2.0`
- Postgres Version `14`
- Storage: `local storage`

## Steps to Reproduce
1. Install the crunchy operator
2. Create a cluster (yaml below)
3. Create a `Route` to the `pgadmin` service
4. Open the `pgadmin` url, login
5. Try to open the database folder on the left

```
apiVersion: postgres-operator.crunchydata.com/v1beta1
kind: PostgresCluster
metadata:
  name: example
  namespace: my-sample-project
spec:
  backups:
    pgbackrest:
      repos:
        - name: repo1
          volume:
            volumeClaimSpec:
              accessModes:
                - ReadWriteOnce
              resources:
                requests:
                  storage: 1Gi
  instances:
    - affinity:
        podAntiAffinity:
          preferredDuringSchedulingIgnoredDuringExecution:
            - podAffinityTerm:
                labelSelector:
                  matchLabels:
                    postgres-operator.crunchydata.com/cluster: hippo-ha
                    postgres-operator.crunchydata.com/instance-set: pgha1
                topologyKey: kubernetes.io/hostname
              weight: 1
      dataVolumeClaimSpec:
        accessModes:
          - ReadWriteOnce
        resources:
          requests:
            storage: 1Gi
      name: pgha1
      replicas: 2
  port: 5432
  postgresVersion: 14
  proxy:
    pgBouncer:
      affinity:
        podAntiAffinity:
          preferredDuringSchedulingIgnoredDuringExecution:
            - podAffinityTerm:
                labelSelector:
                  matchLabels:
                    postgres-operator.crunchydata.com/cluster: hippo-ha
                    postgres-operator.crunchydata.com/role: pgbouncer
                topologyKey: kubernetes.io/hostname
              weight: 1
      port: 5432
      replicas: 2
  userInterface:
    pgAdmin:
      dataVolumeClaimSpec:
        accessModes:
          - ReadWriteOnce
        resources:
          requests:
            storage: 1Gi
      replicas: 1
      service:
        type: ClusterIP
  users:
    - databases:
        - example
      name: pgadmin
```

### EXPECTED

`pgadmin` not throwing errors.

### ACTUAL

`pgadmin` throwing error

1. Describe what actually happens

## Logs

No logs available

## Additional Information

Downgrading the `pgadmin` image to `registry.developers.crunchydata.com/crunchydata/crunchy-pgadmin4:ubi8-4.30-2` fixes the issue

**Comment by tjmoore4** [comment]
@ludydoo This has been addressed with [crunchydata/crunchy-containers#1480](https://github.com/CrunchyData/crunchy-containers/pull/1480) and is planned for inclusion in the next release. Until then, you can downgrade to tag `ubi8-4.30-2` to avoid this issue.

**Comment by ludydoo** [comment]
Thanks for the quick reply 👍
